src/server/logs/cmds/cmds_test.go

```diff
@@ -75,35 +75,36 @@ func TestGetLogs_default_noauth(t *testing.T) {
 	).Run())
 }
 
-func TestGetLogs_default_nonadmin(t *testing.T) {
-	if testing.Short() {
-		t.Skip("Skipping integration tests in short mode")
-	}
+// TODO: need to replace this test with real loki that can filter by data for specific pipelines and non-admin users
```

Comment:
Were these tests failing with the changes? I think that it’s probably better to keep them until we replace them, even if they’re not great.

But if they were incorrectly failing … delete away!